tests/complexity.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var krabbyTest = require('./_baseTest');
+var escomplex = require('escomplex-js');
+var _ = require('lodash');
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+
+var baseConfig = {
+  testConfig: {
+  },
+  files: ['./**/*.js', '!./node_modules/**/*.js', '!./**/node_modules/**/*.js']
+};
+
+var ComplexityTest = new krabbyTest(baseConfig);
+
+ComplexityTest.prototype.analyze = function(files) {
+  var self = this;
+
+  return Promise.map(files, function(fileName) {
+    return fs.readFileAsync(fileName)
```

Comment:
Im not sure if you noticed how many fewer logical loc this file has than jshint.js :trollface: